source/public/styles/styles2.css

```diff
@@ -0,0 +1,415 @@
+}
+
+#calendarContainer{
+    display:flex;
+    justify-content: flex-start;
+    border: 1px solid white;
+    width: 50%;
+    border-radius: 10px;
+}
+
+.calendar{
+    display: flex;
+    flex-direction: column;
+    width: 100%;
+    background-color: white;
+    gap: 20px;
+    border-radius: 10px;
+    
+    
+}
+
+.days{
+    display: flex;
+    width: 97%;
+    margin: 0 auto;
+    justify-content: space-around;
+    border-bottom: 1px solid black;
+    font-weight: bold;
+    background-color: white;
+    padding-bottom:2%;
+    
+}
+
+.day{
+    background-color:white;
+}
+
+.dates{
+    display:grid;
+    grid-template-columns: 1fr;
+    width: 100%;
+   
+    height:100%;
+    
+}
+
+.highlight{
+    background-color: #FFC700 !important;
+    border-radius: 100%;
+
+}
+
+.row{
+    display:grid;
+    grid-template-columns: repeat(7,1fr);
+    a{
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        text-decoration: none;
+    }
+}
+
+.date{
+    background-color: white;
+
+    border:transparent;
+    display:flex;
+    align-items: center;
+    justify-content: center;
+    padding: 20px;
+    font-size: 1.2rem;
+    width: 50%;
+    height:50%;
+    margin:auto;
+    
+
+}
+
+body{
+    display: flex;
+    flex-direction: column;
+    height: 100vh;
+    background: #FFF7EC;
+}
+
+/*.header{
+    display:flex;
+    width:100%;
+    justify-content: center;
+    
+}*/
+.header {
+    
+    height:10%;
+    width: 85%;
+    margin: 0 auto;
+    display: flex;
+    justify-content: space-between; 
+    align-items: center;
+    background-color: white;
+    padding-top: 5%;
+
+}
+
+.headbuttons{
+    display: flex;
+    width:50%;
+    justify-content: space-between;
+    align-items: center;
+    
+}
+.dummyy{
+    display: flex;
+    flex-shrink:1;
+    
+   
+}
+
+.month-year{
+    color: #685489;
+    font-weight:bold;
+    font-size: 2rem;
+    background-color: white;
+    
+
+}
+
+#home-today{
+    border:0.5px solid black;
+    border-radius: 5px;
+    width:12%;
+    height:2.5vh;
+    background-color: white;
+   
+    
+}
+
+.Buttons{
+    border: none; /*gets rid of border around each arrow direction button*/
+    scale: 1.5; /*Sizes the left and right button for the date*/
+    background-color: transparent;
+    
+}
+
+#projects{
+    display: flex;
+    flex-direction: column;
+    border: 1px solid white;
+    width: 25%;
+    background-color: white;
+    gap:20px;
+    border-radius: 10px;
+    
+}
+
+.project-item{
+    width: 80%;
+    margin: 0 auto;
+
+}
+
+.prioritytype{
+    width:75%;
+    background-color:white;
+    border-radius: 32px;
+    color:#FF5353;
+
+}
+
+.edit-project{
+    background-color: transparent;
+    border:none;
+
+
+}
+
+.project-item-delete{
+    
+    background-color: transparent;
+    border:none;
+}
+.fa-trash{
+    color:red;
+    font-size:1.25rem;
+   
+}
+
+.fa-pencil{
+    font-size:1.25rem;
+}
+
+#project-modal{
```

Comment:
Also for the modal, the combined stylesheets should allow you to resuse a lot of the styling rules to avoid repetition